lib/language.js

```diff
@@ -8,7 +8,7 @@ function init() {
   }
 
   language.languages = [
-    { code: 'bg', language: 'Български' }
+    { code: ', language: 'Български' }
     , { code: 'cs', language: 'Čeština' }
     , { code: 'dk', language: 'Dansk' }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2469899589/issues/source?bid=2452379&fileBranchId=3259587#l13)